scipy/fft/__init__.py

```diff
@@ -0,0 +1,90 @@
+"""
+==================================================
+Discrete Fourier transforms (:mod:`scipy.fft`)
+==================================================
```

Comment:
under/over-line length are recommended to be the same. longer is acceptable so I don't think it's your build problem, but same length is better